arm9/source/utils/scripting.c

```diff
@@ -526,6 +528,7 @@ u32 get_flag(char* str, u32 len, char* err_str) {
     if ((len < 2) || (*str != '-')) flag_char = '\0';
     else if (len == 2) flag_char = str[1];
     else if (strncmp(str, "--all", len) == 0) flag_char = 'a';
+    else if (strncmp(str, "--append", len) == 0) flag_char = 'a';
```

Comment:
I hope you made sure there's absolutely no way for `--all` and `--append` to clash. Up to this point I made sure everything uses a letter not used elsewhere.

Reply:
Copy does not use --all, only --append.